src/test/java/software/amazon/nio/spi/s3/S3ClientStoreTest.java

```diff
@@ -166,7 +171,7 @@ public void testGenerateAsyncClientWith403Then301Responses(){
 
     @Test
     public void testGenerateClientWith403Then301ResponsesNoHeader(){
-        // when you get a forbidden response from getBucketLocation
+        // when you get a forbidden response from getBucketLocation --- HERE
```

Comment:
Remove `--- HERE`? Not sure what this is for?

Reply:
a left over to be removed